styles.css

```diff
@@ -163,23 +174,244 @@ button:disabled {
   line-height: 2.5rem;
 }
 
-.project-links {
-  display: flex;
-}
-
 .canopy {
-  color: #344563;
   font-style: normal;
   font-weight: 600;
-  font-size: 1rem;
-  line-height: 1rem;
+  font-size: 0.8rem;
+  color: #344563;
 }
 
 .small-circle {
   width: 0.5rem;
   height: 0.5rem;
   border-radius: 50%;
-  color: #c1c7d0;
+  background-color: #c1c7d0;
+}
+
+.middle-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: 1 90%;
+}
+
+.card {
+  width: 90%;
+  height: 36rem;
+  margin: 1rem auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding: 1rem 1rem 1.5rem 1rem;
+}
+
+.project-image img {
+  width: 100%;
+}
+
+ul.project-links {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding-top: 1.25rem;
+  list-style-type: none;
+}
+
+.project-title {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.4rem;
+}
+
+.career-role,
+.year {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.8rem;
+  line-height: 1rem;
+  color: #7a869a;
+}
+
+.project-description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 0.9rem;
+  line-height: 1.5rem;
+  color: #344563;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.my-skills {
+  display: flex;
+  text-align: center;
+  padding-top: 0.5rem;
+  align-items: center;
+}
+
+.skill-link {
+  margin-right: 1rem;
+  text-decoration: none;
+  padding: 8px;
+  color: #6070ff;
+  background-color: #ebebff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  border-radius: 0.5rem;
+}
+
+a.see-project-btn {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 1.1rem;
+  line-height: 1.5rem;
+  color: #396df2;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background: #fff;
+  border: 1px solid #396df2;
+  padding: 0.5rem;
+  border-radius: 0.8rem;
+  margin-top: 1rem;
+  margin-left: 1rem;
+}
+
+.see-project-btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.see-project-btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.see-project-btn:disabled {
+  background-color: #e5e5e5;
+  color: #5e6c84;
+}
+
+.about-rose {
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding-left: 1rem;
+  padding-top: 4rem;
+  margin-top: 2rem;
+  margin-bottom: -0.6rem;
+  border-top-right-radius: 6rem;
+}
+
+.languages {
+  width: 98%;
+  margin: 0.5rem auto;
+  background-color: #fff;
+  padding-left: 1rem;
+}
+
+.languages-title {
+  color: #000;
+  font-style: normal;
+  font-weight: 500;
+  font-weight: bolder;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+}
+
+/* .language {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  padding-left: 2rem;
+} */
```

Comment:
[OPTIONAL]
- [ ] It's not a good practice to leave commented lines of code in your code base, YOu may consider deleting them for clean code best practice.